roles/matrix-bridge-mx-puppet-skype/defaults/main.yml

```diff
@@ -0,0 +1,205 @@
+# Mx Puppet Skype is a Matrix <-> Skype bridge
+# See: https://github.com/Sorunome/mx-puppet-skype
+
+matrix_mx_puppet_skype_enabled: true
+
+matrix_mx_puppet_skype_container_image_self_build: false
+
+matrix_mx_puppet_skype_docker_image: "sorunome/mx-puppet-skype:latest"
+matrix_mx_puppet_skype_docker_image_force_pull: "{{ matrix_mx_puppet_skype_docker_image.endswith(':latest') }}"
+
+matrix_mx_puppet_skype_base_path: "{{ matrix_base_data_path }}/mx-puppet-skype"
+matrix_mx_puppet_skype_config_path: "{{ matrix_mx_puppet_skype_base_path }}/data"
+matrix_mx_puppet_skype_data_path: "{{ matrix_mx_puppet_skype_base_path }}/data"
+matrix_mx_puppet_skype_docker_src_files_path: "{{ matrix_mx_puppet_skype_base_path }}/docker-src"
+
+matrix_mx_puppet_skype_appservice_port: "6000"
+
+matrix_mx_puppet_skype_homeserver_address: 'http://matrix-synapse:8008'
+matrix_mx_puppet_skype_homeserver_domain: '{{ matrix_domain }}'
+matrix_mx_puppet_skype_appservice_address: 'http://matrix-mx-puppet-skype:{{ matrix_mx_puppet_skype_appservice_port }}'
+
+# A list of extra arguments to pass to the container
+matrix_mx_puppet_skype_container_extra_arguments: []
+
+# List of systemd services that matrix-puppet-skype.service depends on.
+matrix_mx_puppet_skype_systemd_required_services_list: ['docker.service', 'matrix-synapse.service']
+
+# List of systemd services that matrix-puppet-skype.service wants
+matrix_mx_puppet_skype_systemd_wanted_services_list: []
+
+matrix_mx_puppet_skype_appservice_token: ''
+matrix_mx_puppet_skype_homeserver_token: ''
+
+# Can be set to enable automatic double-puppeting via Shared Secret Auth (https://github.com/devture/matrix-synapse-shared-secret-auth).
+matrix_mx_puppet_skype_login_shared_secret: ''
+
+# Default configuration template which covers the generic use case.
+# You can customize it by controlling the various variables inside it.
+#
+# For a more advanced customization, you can extend the default (see `matrix_mx_puppet_skype_configuration_extension_yaml`)
+# or completely replace this variable with your own template.
+matrix_mx_puppet_skype_configuration_yaml: |
+  #jinja2: lstrip_blocks: "True"
+  bridge:
+    # Address for the bridge to bind to; if running as a Docker container, you
+    # probably want 0.0.0.0 here
+    bindAddress: 0.0.0.0
+    # Port to host the bridge on which your homeserver will connect to
+    port: {{ matrix_mx_puppet_skype_appservice_port }}
+    # Name of your homeserver
+    domain: {{ matrix_mx_puppet_skype_homeserver_domain }}
+    # URL where the bridge can connect to your homeserver
+    homeserverUrl: {{ matrix_mx_puppet_skype_homeserver_address }}
+    # Optionally specify a different media URL used for the media store
+    mediaURL: {{ matrix_server_fqn_matrix  }}
+    # This enabled automatic double-puppeting:
+    # A map for shared secrets of the homeserver URL to the shared secret
+    # See https://github.com/devture/matrix-synapse-shared-secret-auth
+    #loginSharedSecretMap:
```

Comment:
Maybe we should have something like this here (accounting for indentation, of course):

```yaml
{% if matrix_mx_puppet_skype_login_shared_secret != '' %}
  loginSharedSecretMap:
    {{ matrix_mx_puppet_skype_homeserver_domain }}: {{ matrix_mx_puppet_skype_login_shared_secret }}
{% endif %}
```

Reply:
FIxed! Added "if" and "endif" in the beginning  of file and the contents indented like the other configuration items.

Reply:
Fixed!